### DISCUSSION TITLE
Outliner Release Announcements and Calls for Translation

### DISCUSSION THREAD

**Discussion by phase1geo** [phase1geo/Outliner#82] [Show and tell]
# What is this discussion board about?

This discussion will act as an announcement board for upcoming Outliner releases and calls for translators to update any necessary translations to go into that release.

**If you are a Outliner translator or have interest in translating Outliner into your language, please follow/subscribe to this discussion thread.**

**Comment by phase1geo** [comment]
Hello everyone!  I am planning on creating a new 1.6.0 stable release for Outliner this Wednesday (9/29/2021).  This is a call for all translators to submit your translations on or before that date.  Happy translating!

**Reply by prokoudine** [reply]
How does one regenerate com.github.phase1geo.outliner.pot and the POT file in po/extra to submit up-to-date translations? I'm guessing you already wrote the new appdata text :)

**Reply by phase1geo** [reply]
Pull the latest master branch.  I had forgotten to add the appdata release information.  I also updated the POTFILES list to include some files that were missing from the list.

**Comment by phase1geo** [comment]
## Call for Outliner 1.7.0 translations!

I am planning on releasing the 1.7.0 release on Monday (5/22/2023). Please have your translation pull requests submitted by 8pm (CST).
